### PR TITLE
No PDFs textview added when list is empty

### PR DESCRIPTION
**Pull request by ishaandhamija**

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/ViewFiles.java [hunk 1]
```diff
@@ -31,6 +29,7 @@
     File folder;
     ListView listView;
     SwipeRefreshLayout swipeView;
+    TextView tv;
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/ViewFiles.java]: Change the name of variable from tv to either textView or emptyStatusTextView

**Comment by ishaandhamija** [comment]
Sure Ma'am :)

**Comment by ishaandhamija** [comment]
TextView name changed ma'am 👍

**Comment by Swati4star** [comment]
Great. I'll merge it soon :D

**Comment by ishaandhamija** [comment]
Thank you so much ma'am :)

**Comment by Swati4star** [comment]
Can you squash these commits into single one.

**Comment by ishaandhamija** [comment]
How can I do that Ma'am :) ?

**Comment by Swati4star** [comment]
You can follow any of the 2 following methods:
[One](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit)
[Two](https://stackoverflow.com/a/5201642/3623128)

**Comment by ishaandhamija** [comment]
Done !!

**Comment by Swati4star** [comment]
Thanks @ishaandhamija 
Merged 👍

**Comment by Swati4star** [comment]
@ishaandhamija there is still one bug here.
If I am on 'View Files' page and there are few PDFs, and I delete all of them one by one, the text view doesn't show up here. IT just shows empty screen.

**Pull request merged**